public/css/style.css

```diff
@@ -89,56 +103,499 @@ a#title {
 #freelancer-home-btn {
   width: 100%;
   height: 65px;
-  background-color: var(--basic-white);
-  color: var(--accent-color);
-  border: 0px;
-  font-size: 1.2em;
+  background-color: var(--accent-color);
+  color: var(--basic-white);
+  border-style: solid;
+  border-width: 0 0 0 1px;
+  border-color: var(--accent-hover);
+  font-size: var(--input-font-size);
   padding: 20px;
 }
 
 #freelancer-home-btn:hover {
-  box-shadow: var(--box-shadow-material-hover);
+  background: var(--accent-hover);
+  cursor: pointer;
+}
+
+#freelancer-home-btn:focus {
+  outline: none;
+}
+
+#react-register {
+  display: flex;
+  align-items: center;
+  margin-right: 20px;
+}
+
+#login-form {
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+#login-username, #login-password {
+  padding: 10px 10px 5px 5px;
+  border: none;
+  border-bottom: solid 2px var(--basic-white);
+  transition: border 0.3s;
+  outline: none;
+  width: 35%;
+  height: auto;
+  font-size: var(--input-font-size);
+  background: var(--accent-color);
+  color: var(--basic-white);
+}
+
+#login-submit, #user-register-btn  {
+  padding-top: 7px;
+  padding-bottom: 7px;
+  width: 80px;
+  height: auto;
+  font-size: var(--input-font-size);
+  border: 0;
+  background-color: var(--basic-white);
+  color: var(--accent-color);
+  border-bottom: solid 4px var(--home-search-border);
+  outline: none;
   cursor: pointer;
+  text-align: center;
+}
+
+#login-submit:hover, #user-register-btn:hover {
+  background-color: var(--background);
+  border-bottom: solid 4px var(--login-button-border);
+}
+
+#login-submit:active, #user-register-btn:active {
+  border-bottom: 0;
+  border-top: solid 4px var(--accent-color);
+}
+
+#user-register-btn {
+  margin-left: 30px;
+}
+
+#topbar-filler-right {
+  flex-grow: 2;
+}
+#topbar-filler-left {
+  flex-grow: 1;
+}
+
+#react-logout {
+  display: flex;
+}
+
+#freelancer-logout-btn {
+  align-self: center;
+  padding-top: 7px;
+  padding-bottom: 7px;
+  width: 80px;
+  height: auto;
+  font-size: var(--input-font-size);
+  border: 0;
+  background-color: var(--basic-white);
+  color: var(--accent-color);
+  border-bottom: solid 4px var(--home-search-border);
+  outline: none;
+  cursor: pointer;
+  text-align: center;
+}
+
+#freelancer-logout-btn:hover {
```

Comment:
Can be grouped toegther with `#freelancer-logout-btn:active` since they have the same declarations